app/javascript/react/screens/App/Overview/__test__/__snapshots__/index.test.js.snap

```diff
@@ -39,7 +41,7 @@ Object {
   "isFetchingClusters": false,
   "isFetchingDatastores": false,
   "isFetchingNetworks": false,
-  "isFetchingProviders": true,
+  "isFetchingProviders": false,
```

Comment:
Hm, it's weird that this snapshot changed `isFetchingProviders` from true to false.. I don't think any of your code changed the providers reducer or any usage of it. Testing locally I see that providers still get loaded just fine when the Overview mounts, so I'll merge anyway.. but do you know why that happened @mzazrivec ?

Reply:
Hmm, actually @mzazrivec for some reason if you include both reducers in the test for `Overview/index.js` this snapshot value goes back to `true`... which it should be, because the test mounts Overview and the action in componentDidMount there should change this state. Idk why adding the productFeaturesReducer there fixes it, maybe we had some silent errors in that snapshot test. But let's pull that in anyway: https://github.com/mturley/manageiq-v2v/commit/60d1ca71c33440cf4e3b0474a6037f25bb05550c

Reply:
You're right, I forgot to include `productFeaturesReducer`, I just added the initial state and ran `yarn test -u`, hence the thing with `isFetchingProviders`. Thanks for the fix, I cherry-picked it to this PR.